basic-examples/basic/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Workflow Description
 
-This workflow simply runs a single step of an example plugin via the default deployer (defined in `config.yaml` as Docker) and reports its success output.
+This workflow simply runs a single step of an example plugin via the default deployer (defined in `config.yaml` as Podman) and reports its success output.
```

Comment:
While it's consistent with the previous text, do you really want to capitalize "Podman" vs using the actual `podman` keyword?